alphastats/gui/utils/analysis_helper.py

```diff
@@ -364,24 +367,22 @@ def helper_compare_two_groups():
     return chosen_parameter_dict
 
 
-def get_sample_names_from_software_file():
+def get_sample_names_from_software_file(loader: BaseLoader) -> List[str]:
```

Comment:
why is this function in the anlaysis helper?

Reply:
moved in https://github.com/MannLabs/alphapeptstats/pull/307